### DISCUSSION TITLE
blog/devlog-25/

### DISCUSSION THREAD

**Discussion by giscus** [modulocity/automationstationgame.com#10] [Comments]
# blog/devlog-25/

Greetings folks! A lot of progress has been made on the game over the last few weeks. From the new tool system to item throwing, and reworked controls, there is a lot to unpack. Let’s dive in!

https://automationstationgame.com/blog/devlog-25/

**Comment by NeanderNerd** [comment]
Great work cuz!! can't wait to play this game into the ground!!
